systemservices/marketplace/src/contracts/manifest.ts

```diff
@@ -1,5 +1,16 @@
+import * as TJS from "typescript-json-schema"
+import { Validator } from "jsonschema"
 import Request from "request-promise-native"
-import { Manifest } from "../types/manifest";
+import { resolve } from "path"
+import { Manifest } from "../types/manifest"
+
+// TODO(ilgooz) create a Definition TS type and generate this schema over it.
+import definitionSchema from '../types/definition.json'
+
+const program = TJS.getProgramFromFiles([resolve(__dirname, '../types/manifest.ts')]);
+const schema = TJS.generateSchema(program, "Manifest", { required: true });
```

Comment:
I think the json schema should be generated once using the command:
```
typescript-json-schema <path-to-typescript-files-or-tsconfig> <type>
```
https://github.com/YousefED/typescript-json-schema


Like we are doing for protobuf files

Reply:
why? dealing with command would be an extra work, an another thing to maintain.

having it programmatically is simpler. schema is created and cached on the start time of the program so, there're no performance drawbacks anyway.

Reply:
What about reusing the json schema in other app?
Having it as source of truth is good anyway.

The code will be simpler and easier to understand.

Reply:
For reusing it somewhere else, it might make sense.

Reply:
Can you please update it.

Reply:
done